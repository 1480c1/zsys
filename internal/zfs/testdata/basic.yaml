pools:
  - name: rpool
    datasets:
      - name: ROOT
      - name: ROOT/ubuntu
        zsys_bootfs: true
        last_used: 2020-09-13T12:26:39+00:00
        mountpoint: /
        canmount: on
