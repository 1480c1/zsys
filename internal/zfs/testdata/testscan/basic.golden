[
   {
      "Name": "TestScan_basic-rpool",
      "Mountpoint": "/",
      "CanMount": "off",
      "BootFS": "",
      "LastUsed": 0,
      "SystemDataset": ""
   },
   {
      "Name": "TestScan_basic-rpool/ROOT",
      "Mountpoint": "/ROOT",
      "CanMount": "off",
      "BootFS": "",
      "LastUsed": 0,
      "SystemDataset": ""
   },
   {
      "Name": "TestScan_basic-rpool/ROOT/ubuntu",
      "Mountpoint": "/",
      "CanMount": "on",
      "BootFS": "yes",
      "LastUsed": 1599999999,
      "SystemDataset": ""
   }
]